libraries/networking/src/udt/PacketList.cpp

```diff
@@ -13,17 +13,26 @@
 
 #include <QDebug>
 
-#include "Packet.h"
+#include "../NLPacket.h"
```

Comment:
I'll make sure, but I don't think I do. There are some asymmetries between NLPacket/Packet and NLPacketList/PacketList, example: PacketList contains `PacketType` but Packet does not (NLPacket does). I think I originally had this in here to do a dynamic cast for reading that information, but I ended up solving it elsewhere.